tasks/test_syntax.js

```diff
@@ -56,7 +60,15 @@ function assertHeaders() {
 
             // parse through code string while keeping track of comments
             var comments = [];
-            falafel(code, {onComment: comments, locations: true}, function() {});
+            falafel(code, {onComment: comments, locations: true}, function(node) {
+                // look for .classList
+                if(node.type === 'MemberExpression') {
+                    var parts = node.source().split('.');
+                    if(parts[parts.length - 1] === 'classList') {
+                        logs.push(file + ' : contains .classList (IE failure)');
```

Comment:
I was hoping eslint would have an option for IE compatibility (actually not even IE11 supports `classList` on SVG elements, apparently)... but I don't see one, so I added it in here. Confirmed that before the change I made to `modebar.js` it failed this test.